composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "paysafegroup/paysafe_sdk_php",
+    "description": "Paysafe PHP SDK",
+    "keywords": ["paysafe","php", "netbanx", "optimalpayments", "sdk"],
+    "license": "MIT",
+    "require": {
+        "php": ">=5.3.0"
+    },
+    "autoload": {
```

Comment:
You're right. I've renamed all files a directories under `source/paysafe` which will fix that. Thanks!